postCompose/src/main/java/com/hamdan/forzenbook/post/compose/Post.kt

```diff
@@ -0,0 +1,83 @@
+package com.hamdan.forzenbook.post.compose
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.hamdan.forzenbook.compose.core.LocalNavController
+import com.hamdan.forzenbook.compose.core.composables.ForzenbookTopAppBar
+import com.hamdan.forzenbook.compose.core.composables.LoginBackgroundColumn
+import com.hamdan.forzenbook.compose.core.composables.PillToggleSwitch
+import com.hamdan.forzenbook.compose.core.composables.PostTextField
+import com.hamdan.forzenbook.compose.core.theme.ForzenbookTheme
+import com.hamdan.forzenbook.ui.core.R
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun PostContent() {
+    val navigator = LocalNavController.current
+    val text = remember { mutableStateOf("") }
+    val focusRequester = remember { FocusRequester() }
+    val keyboard = LocalSoftwareKeyboardController.current
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        topBar = {
+            ForzenbookTopAppBar(topText = stringResource(R.string.top_bar_text_post)) {
+                Image(
+                    painterResource(id = R.drawable.baseline_send_24),
+                    contentDescription = stringResource(id = R.string.post_send_icon),
+                    colorFilter = ColorFilter.tint(ForzenbookTheme.colors.colors.primary),
+                    modifier = Modifier
+                        .padding(ForzenbookTheme.dimens.smallPad_2)
+                        .size(ForzenbookTheme.dimens.iconSizeMedium)
+                        .clickable {},
+                )
+            }
+        },
+        bottomBar = {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = ForzenbookTheme.dimens.smallPad_1)
+            ) {
+                Spacer(modifier = Modifier.weight(1f))
```

Comment:
Replace these spacers with setting the alignment of the Row to be Center

Reply:
ayyy thats truu, the spacers were a leftover from when I was trying to make use of a bottomAppBar composable, it makes more sense for it to just be centered.